waiter/src/waiter/settings.clj

```diff
@@ -276,6 +276,7 @@
                                                 :spnego-auth true}
                                  :force-kill-after-ms 60000
                                  :framework-id-ttl 900000
+                                 :mesos-slave-port 5051
```

Comment:
What was the effect of this not being present?

Reply:
Done.